contrib/Newrelic/Exporter.php

```diff
@@ -0,0 +1,159 @@
+
+    /**
+     * @var string
+     */
+    private $licenseKey;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var SpanConverter
+     */
+    private $spanConverter;
+
+    /**
+     * @var bool
+     */
+    private $running = true;
+
+    /**
+     * @var ClientInterface
+     */
+    private $client;
+
+    public function __construct(
+        $name,
+        string $endpointUrl,
+        string $licenseKey,
+        SpanConverter $spanConverter = null,
+        ClientInterface $client = null
+    ) {
+        $parsedDsn = parse_url($endpointUrl);
+
+        if (!is_array($parsedDsn)) {
+            throw new InvalidArgumentException('Unable to parse provided DSN');
+        }
+        if (
+            !isset($parsedDsn['scheme'])
+            || !isset($parsedDsn['host'])
+            || !isset($parsedDsn['path'])
+        ) {
+            throw new InvalidArgumentException('Endpoint should have scheme, host, port and path');
+        }
+
+        $this->licenseKey = $licenseKey;
+        $this->endpointUrl = $endpointUrl;
+        $this->name = $name;
+        $this->client = $client ?? $this->createDefaultClient();
+        $this->spanConverter = $spanConverter ?? new SpanConverter($name);
+    }
+
+    /**
+     * Exports the provided Span data via the Newrelic protocol
+     *
+     * @param iterable<API\Span> $spans Array of Spans
+     * @return int return code, defined on the Exporter interface
+     */
+    public function export(iterable $spans): int
+    {
+        if (!$this->running) {
+            return Exporter::FAILED_NOT_RETRYABLE;
+        }
+
+        if (empty($spans)) {
+            return Trace\Exporter::SUCCESS;
+        }
+
+        $convertedSpans = [];
+        foreach ($spans as $span) {
+            array_push($convertedSpans, $this->spanConverter->convert($span));
+        }
+        $commonAttributes = ['attributes' => [ 'service.name' => $this->name,
+                                                'host' => $this->endpointUrl, ]];
+        $payload = [[ 'common' => $commonAttributes,
+                     'spans' => $convertedSpans, ]];
+
+        try {
+            $json = json_encode($payload);
+            $headers = ['content-type' => 'application/json',
+                        'Api-Key' => $this->licenseKey,
+                        'Data-Format' => 'newrelic',
+                        'Data-Format-Version' => '1', ];
+            $request = new Request('POST', $this->endpointUrl, $headers, $json);
+            $response = $this->client->sendRequest($request);
+        } catch (RequestExceptionInterface $e) {
+            return Trace\Exporter::FAILED_NOT_RETRYABLE;
+        } catch (NetworkExceptionInterface | ClientExceptionInterface $e) {
+            return Trace\Exporter::FAILED_RETRYABLE;
+        }
+
+        $statusCode = $response->getStatusCode();
+        $reason = $response->getReasonPhrase();
+        echo "\nsendRequest response = " . $statusCode . "\n";
```

Comment:
Thanks for commenting @beniamin! These comments apply to the #275 as well, so I will apply them there  as well :)

That's correct,  I hesitated leaving them in, but initially left them in since it was experimental.  I'll comment out the echos, so when we have logger facilities, we can remember to get them back in.